### PR TITLE
Hathora launch examples UI polishing

### PR DESCRIPTION
**Pull request by jchu231**
- UI Polishing for many of the game examples (Chess, Pong, Avalon, Codenames, Poker, Uno)
- Polish for ObjectDisplay width to be more adaptive based on content
- Polish for ArrayDisplay to including max-height + vertical scrolling and also better UI design to make visual hierarchy more evident to users
- UserId Display polish to handle more complex type of UserId
- Many minor UI papercut type fixes
- Double click handling to copy values

### CODE REVIEW

File: templates/base/client/.rtag/Forms.tsx.hbs [hunk 1]
```diff
@@ -292,7 +292,7 @@ export function Forms({ open, setOpen }: { open: boolean; setOpen: (v: boolean)
           >
             <Dialog.Overlay className="absolute inset-0 transition-opacity bg-gray-500 bg-opacity-75" />
           </Transition.Child>
-          <div className="fixed inset-y-0 right-0 flex max-w-full pl-10">
+          <div className="max-z-index fixed inset-y-0 right-0 flex max-w-full pl-10">
```
Comment (hpx7) [inline templates/base/client/.rtag/Forms.tsx.hbs]: this didn't end up working right?

File: templates/base/client/.rtag/State.tsx.hbs [hunk 2]
```diff
@@ -28,7 +28,9 @@ function KVDisplay(props: { label: string; children: JSX.Element }) {
 }
 
 function ArrayDisplay<T>(props: { value: T[]; children: (value: T) => JSX.Element }) {
-  const [isCollapsed, setIsCollapsed] = useState<boolean>(props.value && props.value.length > 7 ? true : false);
+  const [isCollapsed, setIsCollapsed] = useState<boolean>(props.value &&
+    ((typeof props.value[0] === "object" && props.value.length > 7) ||
+    props.value.length > 4) ? true : false);
```
Comment (hpx7) [inline templates/base/client/.rtag/State.tsx.hbs]: i think the `? true : false` part is unnecessary

File: templates/base/client/.rtag/Forms.tsx.hbs (hunk 1 above)
Comment (jchu231) [reply templates/base/client/.rtag/Forms.tsx.hbs]: Nice catch removed this class + css rules

File: templates/base/client/.rtag/State.tsx.hbs (hunk 2 above)
Comment (jchu231) [reply templates/base/client/.rtag/State.tsx.hbs]: nice catch, cleaned up

**Pull request merged**